NitroxPatcher/Patches/Dynamic/PingInstance_SetVisible_Patch.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Reflection;
+using HarmonyLib;
+using NitroxClient.Communication.Abstract;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Notice the server of a Ping visibility change
+/// </summary>
+public class PingInstance_SetVisible_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((PingInstance t) => t.SetVisible(default));
+
+    public static void Prefix(PingInstance __instance, bool visible)
+    {
+        // Only notice server if there's a change on client-side
+        if (__instance.visible == visible)
+        {
+            return;
+        }
+
+        if (PingInstance_SetColor_Patch.TryGetPingKey(__instance.gameObject, out string pingKey))
```

Comment:
In discord you asked if you can have two patches in one file. What was your final reasoning against that?